backend/api/serializers.py

```diff
@@ -52,3 +52,40 @@ def create(self, validated_data):
 
             else:
                 return False
+
+
+class PlacesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Places
+        fields = [
+            "latitude",
+            "longitude",
+            "radius",
+            "query",
+            "limit",
+            "venues",
+        ]
+        extra_kwargs = {"venues": {"read_only": True}}
+
+    def create(self, validated_data):
+        if self.is_valid():
```

Comment:
How about `early exit` to avoid keeping such a big chunk of code in the if statement?
```suggestion
        if not self.is_valid():
            return None
        
        api_request = PlacesApiRequest(
```